pkg/repo/mysql/services_integration_test.go

```diff
@@ -0,0 +1,213 @@
+package mysql
+
+import (
+	"context"
+	"log"
+	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/google/uuid"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/repo"
+)
+
+var mysqlRepo *Repo
+
+func init() {
+	viper.SetEnvPrefix("translate_mysql")
+	viper.AutomaticEnv()
+
+	mysqlConf := Conf{
+		Addr:     viper.GetString("address"),
+		User:     viper.GetString("user"),
+		Database: viper.GetString("database"),
+	}
+
+	db, err := NewDB(context.Background(), &mysqlConf)
+	if err != nil {
+		log.Panicf("create new db: %v", err)
+	}
+
+	insertQuery := `REPLACE INTO Service (ID, Name)
```

Comment:
You must isolate all tests, no shared data etc. Use helper functions to populate entities as and when needed in tests.